int64 num_points
bool matched
int64 left
int64 right
int64 parent
geometry_msgs/Point mean
geometry_msgs/Point bbox
float64[9] eigenvectors
